Cab share/Cab_shareApp.swift

```diff
@@ -12,16 +12,15 @@ import FirebaseCore
 @main
 struct CabShareApp: App {
     @StateObject var dataController = DataController()
-    
-    init(){
+    @StateObject var loginViewModel = LoginViewModel()
+    init() {
         FirebaseApp.configure()
     }
-
     var body: some Scene {
         WindowGroup {
-            LoginView()
-                .preferredColorScheme(.dark)
+            RoutingView()
```

Comment:
Do we need here PrefferColorScheme?

Reply:
The red part is the one that i deleted with the green one